### PR TITLE
A Compose wrapper for lr scheduler

### PR DESCRIPTION
**Pull request by harryhan618**
This is a first version discussed in #41307 .
A Compose class for lr scheduler like torchvision.transforms.Compose. This Compose is initialized with a list. In the list, each item is a list: [single_lr_scheduler, start_iter_idx, end_iter_idx].

### CODE REVIEW

**Review by vincentqb — CHANGES_REQUESTED** [review]
See [comment](https://github.com/pytorch/pytorch/issues/41307#issuecomment-658448168)
